packages/daisyui/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "extends": "../../tsconfig.base.json",
+    "include": [
+        "src",
+        "../utils/src/types.d.ts"
+    ],
+    "compilerOptions": {
+        "declaration": true,
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
+        "moduleResolution": "Node"
+    },
+    "references": [
+        {
+            "path": "../core"
+        },
+        {
+            "path": "../utils"
+        },
+        {
+            "path": "../validator-ajv8"
+        }
+    ]
+}
```

Adding blank line at end to remove the red circled dash
```suggestion
}

```